script/custom_build_functions.sh

```diff
@@ -1,6 +1,6 @@
 function run_cukes {
   if is_mri_192_plus; then
-    bin/rake acceptance --trace
+    eval "(unset RUBYOPT; exec bin/rake acceptance --trace)"
```

Comment:
Whats this doing?

Reply:
This unset the option we set here:

https://github.com/rspec/rspec-rails/blob/master/script/functions.sh#L16-L18

Reply:
I mostly meant the eval part? But now you mention it why do we have to unset this?

Reply:
See other thread.